src/DIPS.Xamarin.UI/Controls/RadioButtonGroup/RadioButton.xaml

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0"
+      encoding="UTF-8"?>
+<ContentView xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             x:Class="DIPS.Xamarin.UI.Controls.RadioButtonGroup.RadioButton"
+             x:Name="codeBehind">
+    <ContentView.GestureRecognizers>
+        <TapGestureRecognizer Tapped="OnTapped" />
+    </ContentView.GestureRecognizers>
+    <ContentView.Resources>
+        <Thickness x:Key="spacing"
+                   Left="10" />
+    </ContentView.Resources>
+    <Grid BindingContext="{Binding Source={x:Reference codeBehind}}"
+          Padding="{StaticResource spacing}"
+          ColumnSpacing="0">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+
+        <Grid VerticalOptions="Center"
+              HorizontalOptions="Center">
+            <Button x:Name="outerButton"

```

Comment:
To use an `Boxview` style we need three boxes because it does not have a border. This will lead to issues when working with transparent background.

`Frame` corner radius is faulty on iOS (https://github.com/xamarin/Xamarin.Forms/issues/5199). This will be a problem for us if the consumers want to change size.

This is why I used `Button`.